chrome/content/zotero/components/virtualized-table.jsx

```diff
@@ -1090,6 +1090,14 @@ class VirtualizedTable extends React.Component {
 		return this._getVisibleColumns().map((column, index) => {
 			let columnName = formatColumnName(column);
 			let label = columnName;
+			// Allow custom icons to be used in column headers
+			if (column.iconPath) {
+				column.iconLabel = <span>
+				<span
+					className="icon icon-bg"
+					style={{backgroundImage: `url("${column.iconPath}")`}}>
+				</span>&nbsp;{columnName}</span>;
```

Comment:
Is this adding both an icon and a text-based column name?

Reply:
Yes. Should we make the label (column name) optional so that people can create an icon-only column? I guess not. We still need the column name for the column picker.

Reply:
Why would you need both in the header? The only icons we use in the default set are icon-only. Not sure why custom columns wouldn't also be one or the other.

Reply:
Not sure if it is this plugin, I can't remember, but someone does use this kind of column name.

Reply:
We could support passing in a HTML or React node here if we wanted to allow more customization for header cells.

Reply:
> Some plugins, like [Scite], use this icon + text column

Right, but that's just the cell contents. Using both in the header would be extremely rare and does not seem like a particularly good idea. I think we should support icon or text, and maybe design it so that we could add support for custom HTML later if there's demand for it.

Reply:
Done. Support html string or react element as `htmlLabel` now.

When displaying the column header:
htmlLabel > iconPath > iconLabel > label